Gemfile

```diff
@@ -61,6 +61,7 @@ gem 'slim-rails', '~> 3.2'
 gem 'shell-spinner'
 gem 'schema_plus_enums', '~> 0.1'
 gem 'sidekiq', '~> 6.1.3'
+gem 'mimemagic', '~> 0.3.9'
```

Comment:
Nice :)

Reply:
Just wondering is it worth having this upgrade in a separate PR just so its clear that we introduced it? Rather than having it in the same PR as this report change. Just thinking it may be good to know... 🤔